simulations/vip-405/bscmainnet.ts

```diff
@@ -0,0 +1,21 @@
+import { expectEvents } from "../../src/utils";
+import { forking, testVip } from "../../src/vip-framework";
+import vip405 from "../../vips/vip-405/bscmainnet";
+import ACM_COMMANDS_AGGREGATOR_ABI from "./abi/ACMCommandsAggregator.json";
+import ACCESS_CONTROL_MANAGER_ABI from "./abi/AccessControlManager.json";
+import OMNICHAIN_PROPOSAL_SENDER_ABI from "./abi/OmnichainProposalSender.json";
+
+forking(44521298, async () => {
+  testVip("VIP 405 Multichain Governance - Permissions", await vip405(), {
+    callbackAfterExecution: async txResponse => {
+      await expectEvents(
+        txResponse,
+        [OMNICHAIN_PROPOSAL_SENDER_ABI],
+        ["ExecuteRemoteProposal", "StorePayload"],
+        [5, 0],
+      );
+      await expectEvents(txResponse, [ACCESS_CONTROL_MANAGER_ABI], ["RoleGranted"], [3]);
+      await expectEvents(txResponse, [ACM_COMMANDS_AGGREGATOR_ABI], ["GrantPermissionsExecuted"], [1]);
+    },
+  });
```

Comment:
We could add the check to verify that the DEFAULT_ROLE has been revoked for ACMAggregator

Reply:
[1de5938](https://github.com/VenusProtocol/vips/pull/431/commits/1de593847ad638ceb24ac74e89d914eaa64e3d97)